assets/sass/_vars/_colors.scss

```diff
@@ -1 +1,6 @@
-$logo-color: #e23838;
\ No newline at end of file
+$logo-color: #e23838;
+$logo-second-color: #fffff;
```

Comment:
This one is not required anymore, I think :)

Reply:
let's change it to a bit off white

Reply:
Yes dai, I will assign blue color (as per ruby nepal's logo color) and can be used for the links of the blog title.  ```$logo-second-color: #fffff;``` this is not used anywhere yet.